src/functionalTest/java/uk/gov/hmcts/reform/sscscorbackend/SscsCorBackendRequests.java

```diff
@@ -190,7 +190,10 @@ public void uploadHearingEvidence(String hearingId, String fileName) throws IOEx
 
     public void submitHearingEvidence(String hearingId, String description) throws IOException {
         HttpResponse response = postRequest("/continuous-online-hearings/" + hearingId + "/evidence",
-                new StringEntity("{\"body\":\"" + description + "\"}", APPLICATION_JSON)
+            new StringEntity("{\n"
+                + "  \"body\": \"" + description + "\",\n"
+                + "  \"idamEmail\": \"mya-sscs-6920-reps@mailinator.com\"\n"
```

Comment:
Please remove this had coded values

Reply:
it's a dummy value that we need in the functional test. Why do you want to remove it?